internal/transport/http2_client.go

```diff
@@ -1265,7 +1265,11 @@ func (t *http2Client) handleGoAway(f *http2.GoAwayFrame) {
 		return
 	}
 	if f.ErrCode == http2.ErrCodeEnhanceYourCalm {
```

Comment:
Perhaps add the too many pings conditional here, corresponding to the and in this line: with error code ENHANCE_YOUR_CALM "and" debug data equal to ASCII "too_many_pings"

Reply:
done